presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -221,22 +221,21 @@ public void testReadFullAcidWithOriginalFiles(boolean isPartitioned, BucketingTy
             onHive().executeQuery("INSERT INTO TABLE " + tableName + hivePartitionString + " VALUES (22, 2)");
             onHive().executeQuery("ALTER TABLE " + tableName + " SET " + hiveTableProperties(ACID, bucketingType));
 
-            String selectPrestoQuery = "SELECT col, fcol FROM " + tableName + " ORDER BY col";
             // read with original files
-            assertThat(query(selectPrestoQuery)).containsOnly(row(21, 1), row(22, 2));
```

Comment:
Producr tests were failing because of the different order of row, that was the motive to add ORDER BY.

Reply:
nevermind just saw the change of replacing `containsOnly`. looks fine. thanks.